lib/test/fixtures/getDocumentsFixture.js

```diff
@@ -19,11 +17,11 @@ function getDocumentsFixture() {
   );
 
   return [
-    factory.create(contract, userId, 'niceDocument', { name: 'Cutie' }),
-    factory.create(contract, userId, 'prettyDocument', { lastName: 'Shiny' }),
-    factory.create(contract, userId, 'prettyDocument', { lastName: 'Sweety' }),
+    factory.create(contract, contract.getId(), 'niceDocument', { name: 'Cutie' }),
```

Comment:
The second argument is `userId`, so I would keep both: contract from the fixture and a generated random id for the user.